h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderModel.java

```diff
@@ -33,6 +33,9 @@ public TargetEncoderModel(Key<TargetEncoderModel> selfKey, TargetEncoderParamete
   public static class TargetEncoderParameters extends Model.Parameters {
     public boolean _blending = false;
     public BlendingParams _blending_parameters = TargetEncoder.DEFAULT_BLENDING_PARAMS;
+    public double _k = TargetEncoder.DEFAULT_BLENDING_PARAMS.getK();
```

Comment:
@honzasterba right, this is just because random grid search will not be able to work with `BlendingParams` type of `_blending_parameters` field as it is kind of hierarchical structure of hyper parameters. Just for the sake of testing in this PR I added(exposed) these two parameters. I have not removed the `_blending_params` itself because it is done in another PR https://github.com/h2oai/h2o-3/pull/4008 , so I will rebase later and that is it

Reply:
@honzasterba regarding naming - `k` and `f` names come from the original paper for Target Encoding. But actually we were already using more meaningful naming for them `inflection_point` and `smoothing` - but in last version of TE's api it was changed back for unknown for me reasons. I will consider to change naming later but it is a separate PR - poor users... so many changes in API :(

Reply:
your reply makes me even more sure that this should be removed, its technically part of java API and we should be very careful about adding things here without proper reason (testing something completely unrelated is not a proper reason)